config-model/src/main/java/com/yahoo/vespa/model/container/Container.java

```diff
@@ -139,10 +131,15 @@ public Http getHttp() {
         return (parent instanceof ContainerCluster) ? ((ContainerCluster) parent).getHttp() : null;
     }
 
-    public JettyHttpServer getDefaultHttpServer() {
```

Comment:
getDefaultHttpServer is used in YahooContainerModelAmender and currently fails the build.
[0] honda:~/git/vespa-yahoo$ git grep getDefaultHttpServer
config-model-fat-amended/src/main/java/com/yahoo/vespa/model/container/amender/YahooContainerModelAmender.java:                .orElseGet(container::getDefaultHttpServer);